string classname
uint8  classID
geometry_msgs/Point center_point
geometry_msgs/Point32[] point_cloud